diff --git a/node_modules/react-native-sound/RNSound.podspec b/node_modules/react-native-sound/RNSound.podspec
index 45fd642..5f96c56 100644
--- a/node_modules/react-native-sound/RNSound.podspec
+++ b/node_modules/react-native-sound/RNSound.podspec
@@ -12,7 +12,7 @@ Pod::Spec.new do |s|
   s.source              = { :git => 'https://github.com/zmxv/react-native-sound.git', :tag => "v#{s.version}" }
   s.default_subspec     = 'Core'
   s.requires_arc        = true
-  s.platform            = :ios, "7.0"
+  s.platforms           = { :ios => "10.0", :tvos => "12.0" }
 
   s.dependency 'React'
 
diff --git a/node_modules/react-native-sound/RNSound/RNSound.m b/node_modules/react-native-sound/RNSound/RNSound.m
index 5950dfe..33207f5 100644
--- a/node_modules/react-native-sound/RNSound/RNSound.m
+++ b/node_modules/react-native-sound/RNSound/RNSound.m
@@ -351,10 +351,13 @@ RCT_EXPORT_METHOD(getCurrentTime
 
 RCT_EXPORT_METHOD(setSpeakerPhone : (BOOL)on) {
     AVAudioSession *session = [AVAudioSession sharedInstance];
+#if !TARGET_OS_TV
     if (on) {
         [session overrideOutputAudioPort:AVAudioSessionPortOverrideSpeaker
                                    error:nil];
-    } else {
+    } else
+#endif
+    {
         [session overrideOutputAudioPort:AVAudioSessionPortOverrideNone
                                    error:nil];
     }
