--- node_modules/react-native/Libraries/Utilities/HMRClient.js	2018-01-31 00:06:48.000000000 -0800
+++ node_modules/react-native/Libraries/Utilities/HMRClient.js	2018-01-31 00:07:02.000000000 -0800
@@ -15,7 +15,7 @@
 const Platform = require('Platform');
 const invariant = require('fbjs/lib/invariant');
 
-const MetroHMRClient = require('metro/src/lib/bundle-modules/HMRClient');
+const MetroHMRClient = require('metro/src/lib/bundle-modules/HmrClient');
 
 /**
  * HMR Client that receives from the server HMR updates and propagates them
